task_DEV_6/task_DEV_6/Stack.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+
+namespace task_DEV_6
+{
+    class Stack
+    {        
+        public int[] stack;
+        public int count;
+
+        public Stack()
+        {
+            stack = new int[10];
+        }
+        public Stack(int number)
```

Comment:
Where is empty line between constructors?

Reply:
`number ` bad name.

Reply:
Done.